skbeam/core/accumulators/binned_statistic.py

```diff
@@ -434,36 +474,21 @@ def __init__(self, rbins, phibins, rowsize, colsize,
                 will be called on the values in each bin.  Empty bins will be
                 represented by function([]), or NaN if this returns an error.
         """
+        rpix, phipix = get_r_phi(shape, origin)
```

Comment:
Thanks very much for the great suggestion, Julien, and my apologies for taking some days to reply (multitasking, as usual).  I tried the above, and encountered two things that I needed to adapt:  I had to add an "indexing" option to radial_grid because I need to use 'ij' indexing of meshgrid, and then I had to swap the rows/columns, so my code ended up looking like this:

        rpix = radial_grid((colc,rowc), (shape[1],shape[0]), indexing='ij')

I'm happy to do it this way, although I would have to add "indexing='xy'" option as the default to utils.py.  Or I could do it the way I currently am.  Anyone have any preferences?